proto/xudrpc.proto

```diff
@@ -218,7 +220,7 @@ message CancelOrderRequest {
 }
 
 message CancelOrderResponse {
-  string result = 1 [json_name = "result"];
+  bool result = 1 [json_name = "result"];
```

Comment:
Let's rename this to `canceled` so that it's more clear what the response means - true means an order was canceled and false means it wasn't. We can put a comment in that says that too.